inc/core/deprecated/deprecated-filters.php

```diff
@@ -85,3 +85,105 @@ function astra_apply_filters_deprecated( $tag, $args, $version, $replacement = f
 		}
 	}
 }
+
+// Deprecating ast_footer_bar_display filter.
+add_filter( 'astra_footer_bar_display', 'astra_deprecated_ast_footer_bar_display_filter', 10, 1 );
+
+/**
+ * Display footer builder markup.
+ *
+ * @since x.x.x
+ * @param boolean $display_footer true | false.
+ * @return boolean $display_footer true for enabled | false for disable.
+ */
+function astra_deprecated_ast_footer_bar_display_filter( $display_footer ) {
+
+	$display_footer = astra_apply_filters_deprecated( 'ast_footer_bar_display', array( $display_footer ), '3.7.4', 'astra_footer_bar_display', '' );
+
+	return $display_footer;
```

Comment:
```suggestion
	return astra_apply_filters_deprecated( 'ast_footer_bar_display', array( $display_footer ), '3.7.4', 'astra_footer_bar_display', '' );
```

Extra variable creation can be avoided here